test/test_fs.py

```diff
@@ -142,6 +149,17 @@ def test_attr_timeout(testfs):
         os.fstat(fh.fileno())
         assert fs_state.getattr_called
 
+def test_call_stop_from_another_thread(testfs):
+    (mnt_dir, fs_state, need_stop) = testfs
+    path = os.path.join(mnt_dir, 'message')
+    os.stat(path)
+    assert fs_state.lookup_called
+    fs_state.lookup_called = False
+    os.stat(path)
+    assert not fs_state.lookup_called
+
+    need_stop.set()
```

Comment:
It seems to me that you really only need the first and last line of this function. What is the purpose of the others?

Reply:
I agree, the fs accesses are not strictly needed but make sure the fs is up and running (i.e. not in a "process-just-spawned" state). I can remove this if you think it's useless.

Reply:
Makes sense, but let's just use a single `os.stat` call and add a "Ensure FS is ready" comment.

Reply:
done